docs/quickstart_guide.md

```diff
@@ -207,6 +243,18 @@ Execute the following below **as a non-root user with sudo rights** using **bash
     ```bash
     ./scripts/cloudlab/setup_node.sh;
     ```
+    > **Note:**
+    > To enable runs with `stargz` images, setup kubelet by adding the `stock-only` and `use-stargz`
+    > flags as follows:
+    >   ```bash
+    >   ./scripts/cloudlab/setup_node.sh stock-only use-stargz
+    >   ```
+    > **IMPORTANT**
+    > Currently `stargz` is only supported in native kubelet contexts without firecracker. 
+    > Therefore, the following steps from this guide must **not** be executed:
+    > * `2.3`,
```

Comment:
I would add the titles of the subsections, as it can be a little confusing just giving the number. It would be a bit more clear